tests/framework/utils/k8s_helper.go

```diff
@@ -87,7 +95,12 @@ func CreateK8sHelper(t func() *testing.T) (*K8sHelper, error) {
 	return h, err
 }
 
-var k8slogger = capnslog.NewPackageLogger("github.com/rook/rook", "utils")
+var (
+	k8slogger  = capnslog.NewPackageLogger("github.com/rook/rook", "utils")
+	kubectlCmd = GetCmd()
```

Comment:
kubectlCmd needs to be updated as well as it's not always kubectl

Reply:
@Madhu-1  `CliCmd` or just `Cmd` ?

Reply:
Clicmd sounds good

Reply:
Done.